scripts/travis_test.sh

```diff
@@ -143,7 +143,7 @@ if [ "$should_run_tests" = true ]; then
     echo "Measuring code coverage..."
     measure_cov "manticore/core/smtlib/*" 80
     measure_cov "manticore/core/cpu/x86.py" 50
-    measure_cov "manticore/core/memory.py" 85
```

Comment:
Leaving comment here, but it's about travis in general. We should only install `[native]` for linux/cpu tests and only install `[evm]` for evm tests to make sure the delineation is clear and complete.